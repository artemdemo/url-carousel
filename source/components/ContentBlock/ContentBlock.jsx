import React from 'react';

import './ContentBlock.less';

const ContentBlock = (props) => {
    return (
        <div className='content-block'>
            {props.children}
        </div>
    );
};

export default ContentBlock;
