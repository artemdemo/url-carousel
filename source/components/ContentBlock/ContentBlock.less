.content-block {
    margin-bottom: 10px;
}
