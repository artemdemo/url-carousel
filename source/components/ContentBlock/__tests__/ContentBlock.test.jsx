import React from 'react';
import renderer from 'react-test-renderer';
import ContentBlock from '../ContentBlock';

describe('<ContentBlock>', () => {
    it('Render empty', () => {
        const tree = renderer.create(
            <ContentBlock />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('Render with content', () => {
        const tree = renderer.create(
            <ContentBlock>
                Some text
            </ContentBlock>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
