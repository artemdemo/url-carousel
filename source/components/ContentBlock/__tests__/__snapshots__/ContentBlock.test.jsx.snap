exports[`<ContentBlock> Render empty 1`] = `
<div
  className="content-block" />
`;

exports[`<ContentBlock> Render with content 1`] = `
<div
  className="content-block">
  Some text
</div>
`;
